sdks/urbackend-python/test_sdk.py

```
@@ -0,0 +1,371 @@
+    separator("2. Sign Up")
+    try:
+        result = client.auth.sign_up(EMAIL, PASSWORD, username="sdk_tester")
+        print(f"✅ Signed up: {result}")
+    except (AuthError, ValidationError) as e:
+        print(f"ℹ️  Sign-up skipped (user likely exists): {e.message}")
+
+    # ── 3. LOGIN ────────────────────────────────────────────────
+    separator("3. Login")
+    try:
+        session = client.auth.login(EMAIL, PASSWORD)
+        print(f"✅ Login successful!")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)